rest-api-spec/src/main/resources/rest-api-spec/test/search.aggregation/370_multi_terms.yml

```diff
@@ -618,3 +618,51 @@ setup:
   - match: { aggregations.m_terms.buckets.0.key: ["a", 1] }
   - match: { aggregations.m_terms.buckets.0.key_as_string: "a|1" }
   - match: { aggregations.m_terms.buckets.0.doc_count: 4 }
+
+---
+"aggregate over multi-terms test":
+  - skip:
+      version: "- 2.0.99"
+      reason:  multi_terms aggregation is introduced in 2.1.0
```

Comment:
The mixed cluster tests will run this against the version on the 2.x branch, which is 2.8.0. I'm assuming 2.8 will fail this test without your change, which means the version here should be `2.9.99` to prevent it from running against anything older than 3.0. Please update the reason text as well.

I think this change should be backported (it is a bug fix/enhancement, not a breaking change, right?). Once it is backported, we can update the version guard here to be 2.7.99 since it will be safe to run against 2.8 after the backport. (I know all of this is complicated...)

Reply:
This is a bug fix, not a breaking change.

Reply:
Can you have a look at the latest diff?  Thanks.

Reply:
@andrross you good with this? conversation needs resolving to merge ;)

Reply:
@andrross ?